spr/src/output.rs

```diff
@@ -13,9 +13,16 @@ pub fn output(icon: &str, text: &str) -> Result<()> {
     let bullet = format!("  {}  ", icon);
     let indent = console::measure_text_width(&bullet);
     let indent_string = " ".repeat(indent);
-    let options = textwrap::Options::new((term.size().1 as usize) - indent * 2)
-        .initial_indent(&bullet)
-        .subsequent_indent(&indent_string);
+    let mut options =
+        textwrap::Options::new((term.size().1 as usize) - indent * 2)
+            .initial_indent(&bullet)
+            .subsequent_indent(&indent_string);
+
+    if lazy_regex::regex_is_match!(r#"https?://"#, text) {
+        options.break_words = false;
+        options.word_separator = textwrap::WordSeparator::AsciiSpace;
+        options.word_splitter = textwrap::WordSplitter::NoHyphenation;
+    }
```

Comment:
These options look reasonable. I'd be okay with using them unconditionally.

```suggestion
        options.break_words = false;
        options.word_separator = textwrap::WordSeparator::AsciiSpace;
        options.word_splitter = textwrap::WordSplitter::NoHyphenation;
```

Reply:
sounds good! just updated the diff